### DISCUSSION TITLE
Add support media from CDN as url

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#52] [Feature Requests]
Hello. I now want to generate products locally in the database. Each product has its own link image. Since now this is not possible with your platform, I would like to propose an idea to create such an opportunity to use a link as a media from any website or CDN. I generate products using seeders

**Comment by glennjacobs** [comment]
Am I correct in saying you want to be able to download an image from a URL and associate it with a product?

If you're not wanting to use the built-in image system, then another approach would be to create a custom attribute to store your image URL.

**Reply by devmansurov** [reply]
Yes. Currently I've added attribute like this:
```
Attribute::create([
    'attribute_type'     => ProductType::class,
    'attribute_group_id' => $group->id,
    'position'           => 3,
    'name'               => [
        'en' => 'Image URL',
        'ru' => 'Ссылка картинки',
    ],
    'handle'        => 'image_cdn',
    'section'       => 'main',
    'type'          => Text::class,
    'required'      => false,
    'default_value' => null,
    'configuration' => [
        'type' => 'text',
    ],
    'system' => false,
]);
```

But this attribute can contain only 1 image. I need to array of images from CDN. I can also add images as json but array field type will be cool.
